plugins/modules/proxmox_backup_schedule.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025 Marzieh Raoufnezhad <raoufnezhad@gmail.com>
+# Copyright (c) 2024 Maryam Mayabi <mayabi.ahm at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: proxmox_backup_schedule
+
+short_description: Scheduling VM Backups and Removing it.
+
+version_added: 10.3.0
+
+description: The proxmox_backup_schedule module modify backup jobs such as set or delete vmid.
+
+author:
+  - "Marzieh Raoufnezhad (@raoufnezhad) <raoufnezhad@gmail.com>"
+  - "Maryam Mayabi (@mmayabi) <mayabi.ahm@gmail.com>"
+
+options:
+  vm_name:
+    description:
+      - The name of the Proxmox VM.
+      - Mutually exclusive with O(vm_id).
+    required: false
+    type: str
+  vm_id:
+    description:
+      - The ID of the Proxmox VM.
+      - Mutually exclusive with O(vm_name).
+    required: false
+    type: str
+  backup_id:
+    description: The backup job ID.
+    required: false
+    type: str
+  backup_action:
```

Comment:
This kind of parameter that controls the action being taken by the module, in Ansible is usually named `state`. I do not see that as mandatory, but you might want to consider it.